README.md

```diff
@@ -126,4 +154,5 @@ Feedstock Maintainers
 * [@dzelge](https://github.com/dzelge/)
```

Comment:
```suggestion
```

Reply:
Thanks, good catch! I removed dzelge, added Tejas, and sorted alphabetically.